pages/user/[id].vue

```diff
@@ -104,30 +112,59 @@
             <div class="primary-stat">
               <DownloadIcon class="primary-stat__icon" aria-hidden="true" />
               <div class="primary-stat__text">
-                <span class="primary-stat__counter">{{ sumDownloads }}</span>
-                downloads
+                <IntlFormatted
+                  :message-id="messages.profileDownloadsStats"
+                  :values="{ count: sumDownloads }"
```

Comment:
`formatCompactNumber` should be used here (retrieved using `useCompactNumber` composable). [Example](https://vintl.vercel.app/guide/extras/compact-numbers.html#using-compact-numbers)

Reply:
Why is it necessary to use `formatCompactNumbe` ?

Reply:
It allows to display numbers in compact notation (e.g. 1,200 will be displayed as 1.2K in English and 1.200 in German). However, standard APIs aren't good at this, and that's [why `@vintl/compact-number` was created](https://github.com/vintl-dev/compact-number#prehistory).

It's already in Knossos, you just gotta get the formatter function using `useCompactNumber` composable, and then use it on `sumDownloads` here. 

You will also have to make sure that `count` plural doesn't use compound character `#` in plural cases, or otherwise it will be replaced with incorrect number. Instead, you should use the argument directly:

```
{count, plural, one {<stat>{count}</stat> download} other {<stat>{count}</stat> downloads}}
```

Reply:
So, if I understand, It's required for me to create a plugin before using `formatCompactNumber` function ?

Reply:
No, it's already in Knossos — https://github.com/modrinth/knossos/blob/57964282ee52d5689dbb9a53a62d6681fdabb2a8/composables/compact-number.ts.

Just add the following to the `<script setup>`:
```js
const formatCompactNumber = useCompactNumber()
```

Reply:
Oh damn, I'm so stupid :/